lesson02/assignment/README.md

```diff
@@ -1,24 +1,31 @@
 # Assignment 02 - Sass/Gulp/npm
 
 ### Required setup
-- Install version 13 of [node](https://nodejs.org/en/) (this should be the latest version)
+- Install [Adobe XD](https://www.adobe.com/products/xd.html) (free)
+- Install [node](https://nodejs.org/en/)
+*A note on Node:* We have some compatibility constraints in the starter code that requires Node v10 be used locally. When you install Node on your system (necessary for all assignments and activities after this), please install version 10. There are two ways that this can be done:
+
+    1. Download the correct version from Node files here: https://nodejs.org/dist/latest-v10.x/ The "darwin" files are for Mac, "win" for Windows.
+    2. Use Node Version Manager to install and use multiple versions of Node on your system. https://github.com/nvm-sh/nvm This will be more extensible for future use, but is a little more complex to get started with.
+
+If you've already installed a more recent version of Node locally, it should be uninstalled before installing v10.
 
 ### To get started:
 -	In your forked repo, create a new branch off of master called 'lesson-02'
 -   Navigate to the root folder of this assignment within terminal `lesson02/assignment/`
 -   Run `npm install`
 -   Run `npx gulp`
-  -   Expected output from gulp is information about localhost addresses, and a browser window opening to http://localhost:3000
+  -   Expected output from gulp is information about localhost addresses, and a browser window opening to http://localhost:3000.
+  - If you instead see error output (like "primordials is not defined"), ensure you're using Node v10.
+  -	If you see permissions errors, this [link](https://stackoverflow.com/questions/50258901/cannot-find-module-gulp-sass) has information that may help you resolve.
 -   Leave gulp running. As you work with the files, browsersync will watch your scss and HTML files and reload the page in the browser on changes.
-- Find the mockup of the component in the `ref` folder
-- Build the component using scss, and follow the requirements found in the assignment description
-- Make sure your implementation is responsive. Feel free to make changes to the design as necessary for it to look good on small screens.
+- Find the XD/JPG of the mockup component in the `ref` folder.
+- Build the component using scss, and follow the requirements found in canvas.
+- Make sure to have the image/title stack on top of the rest of the card info on mobile
 
 ### To Submit:
 - Create a Pull Request for your assignment branch to your forked master
 - Submit the link to your pull request in your assignment submission box
 - Once assignment is graded, merge the Pull Request
 
-#### Image credit:
-
-"handsome" by foreverdigital is licensed with CC BY-NC-ND 2.0. To view a copy of this license, visit https://creativecommons.org/licenses/by-nc-nd/2.0/
+#### Image source: https://unsplash.com/photos/OhKElOkQ3RE
```

Comment:
It looks like this MR is including changes from a previous version of this repo. Could I ask you to pull down the latest master locally, and merge & push it both on master branch and this assignment branch, before merging this MR? I don't want your repo to get in a weird git state. If you didn't add upstream as a remote yet, do that first. In the example below, I use the SSH link, but if you use HTTPS links to repos, use that instead (https://github.com/UWFront-End-Cert/html300-v2.git).

```
$ git remote add upstream git@github.com:UWFront-End-Cert/html300-v2.git
$ git checkout master
$ git pull upstream master 
$ git push origin master
$ git checkout assignment02
$ git merge master
$ git push origin assignment02
```

If you have any questions on this or get stuck, email me or drop into office hours and we'll work through it together.

Reply:
Now that this is graded, you can merge it. Do that before following steps above.